lms/services/ltia_http.py

```diff
@@ -15,6 +15,21 @@ def __init__(
         self._jwt_service = jwt_service
         self._http = http
 
+    def sign(self, payload: dict):
```

Comment:
This becomes its own method.  The default for "aud" here is the issuer.

Reply:
Doc strings again